### ISSUE TITLE
Doesn't recognize that I have Tunnelblick installed

### ISSUE THREAD

**Issue by oalders** [deanishe/alfred-vpn-manager#6]
When I try to run the workflow I get a message that: `Application "Viscosity" is not installed`.  I'm actually just interested in managing Tunnelblick, but that doesn't seem to be an option.

![image](https://user-images.githubusercontent.com/96205/35244860-a8e3c620-ff8f-11e7-8a08-8cee524c9b8c.png)

![image](https://user-images.githubusercontent.com/96205/35244909-cb678830-ff8f-11e7-96a9-797e83603aa7.png)

```
[2018-01-22 16:13:39][STDERR: input.scriptfilter] .
16:13:39 workflow.py:2055 DEBUG    ---------- VPN Manager (3.0.1) ----------
16:13:39 workflow.py:1462 DEBUG    reading settings from /Users/olaf/Library/Application Support/Alfred 3/Workflow Data/net.deanishe.alfred.vpn.manager/settings.json
16:13:39 workflow.py:2339 DEBUG    update check not due
16:13:39 vpn.py:523 DEBUG    args : {'--all': False,
 '--help': False,
 '<name>': None,
 '<query>': u'',
 'conf': False,
 'connect': False,
 'disconnect': False,
 'list': True}
16:13:39 vpn.py:120 DEBUG    appinfo=None
16:13:39 workflow.py:2248 DEBUG    set last run version: 3.0.1
16:13:39 workflow.py:2097 DEBUG    ---------- finished in 0.058s ----------
```

```
{
  "__workflow_last_version": "3.0.1"
}
```

**Comment by deanishe** [comment]
What version of macOS are you running?

Could you post the correct log? That's the log from the first screenshot, where the workflow is behaving as expected (Viscosity *isn't* installed).

To help you, I need the log from the second screenshot where the actual error is happening.

**Comment by oalders** [comment]
Ah, ok.  The first one seemed like an error to me as well because it said nothing about Tunnelblick.

MacOS 10.13.3

```
[2018-01-29 16:07:41][input.scriptfilter] Queuing argument ''
[2018-01-29 16:07:41][input.scriptfilter] Script with argument '' finished
[2018-01-29 16:07:41][STDERR: input.scriptfilter] .
16:07:41 workflow.py:2055 DEBUG    ---------- VPN Manager (3.0.1) ----------
16:07:41 workflow.py:1462 DEBUG    reading settings from /Users/olaf/Library/Application Support/Alfred 3/Workflow Data/net.deanishe.alfred.vpn.manager/settings.json
16:07:41 workflow.py:2339 DEBUG    update check not due
16:07:41 vpn.py:523 DEBUG    args : {'--all': False,
 '--help': False,
 '<name>': None,
 '<query>': u'',
 'conf': False,
 'connect': False,
 'disconnect': False,
 'list': True}
16:07:41 vpn.py:120 DEBUG    appinfo=None
16:07:41 workflow.py:2248 DEBUG    set last run version: 3.0.1
16:07:41 workflow.py:2097 DEBUG    ---------- finished in 0.058s ----------
[2018-01-29 16:07:41][input.scriptfilter] {"items": [{"valid": false, "subtitle": "Use \"vpnconf\" to change the application", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertCautionIcon.icns"}, "title": "Application \"Viscosity\" is not installed"}]}
[2018-01-29 16:07:43][input.scriptfilter] Queuing argument ''
[2018-01-29 16:07:43][input.scriptfilter] Script with argument '' finished
[2018-01-29 16:07:43][ERROR: input.scriptfilter] Code 1: .
16:07:43 workflow.py:2055 DEBUG    ---------- VPN Manager (3.0.1) ----------
16:07:43 workflow.py:1462 DEBUG    reading settings from /Users/olaf/Library/Application Support/Alfred 3/Workflow Data/net.deanishe.alfred.vpn.manager/settings.json
16:07:43 workflow.py:2339 DEBUG    update check not due
16:07:43 vpn.py:523 DEBUG    args : {'--all': False,
 '--help': False,
 '<name>': None,
 '<query>': u'',
 'conf': True,
 'connect': False,
 'disconnect': False,
 'list': False}
16:07:43 workflow.py:1689 DEBUG    loading cached data: /Users/olaf/Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Data/net.deanishe.alfred.vpn.manager/__workflow_update_status.cpickle
16:07:43 workflow.py:2269 DEBUG    update_data: {u'available': False}
16:07:43 vpn.py:120 DEBUG    appinfo=AppInfo(name=u'Tunnelblick', path=u'/Applications/Tunnelblick.app', bundleid=u'net.tunnelblick.tunnelblick')
16:07:43 vpn.py:120 DEBUG    appinfo=None
16:07:43 workflow.py:2074 ERROR    'NoneType' object has no attribute 'path'
Traceback (most recent call last):
  File "/Users/olaf/Library/Application Support/Alfred 3/Alfred.alfredpreferences/workflows/user.workflow.8F40B5E9-CE65-48F0-A629-2C318756271B/workflow/workflow.py", line 2067, in run
    func(self)
  File "./vpn.py", line 535, in main
    return do_config(args.get('<query>'))
  File "./vpn.py", line 343, in do_config
    icon=app.info.path,
AttributeError: 'NoneType' object has no attribute 'path'
16:07:43 workflow.py:2097 DEBUG    ---------- finished in 0.119s ----------
[2018-01-29 16:07:43][input.scriptfilter] {"items": [{"valid": false, "subtitle": "'NoneType' object has no attribute 'path'", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertStopIcon.icns"}, "title": "Error in workflow 'VPN Manager'"}]}
```

**Comment by deanishe** [comment]
Thanks. So it *is* finding Tunnelblick just fine. Now I know where to look for the issue.

**Comment by deanishe** [comment]
As a temporary fix, if you open [the workflow configuration sheet](https://www.alfredapp.com/help/workflows/advanced/variables/#environment) in Alfred Preferences, and change `VPN_APP` to `Tunnelblick`, it will work.

**Comment by deanishe** [comment]
Fixed in [v3.0.2](https://github.com/deanishe/alfred-vpn-manager/releases/tag/v3.0.2).

**Comment by oalders** [comment]
Yep, that works.  Thanks @deanishe!
